.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -51,7 +51,8 @@ data class CIBuildModel (
                             TestCoverage(TestType.allVersionsCrossVersion, OS.windows, JvmVersion.java7),
                             TestCoverage(TestType.noDaemon, OS.linux, JvmVersion.java8),
                             TestCoverage(TestType.noDaemon, OS.windows, JvmVersion.java8),
-                            TestCoverage(TestType.platform, OS.macos, JvmVersion.java8)),
+                            TestCoverage(TestType.platform, OS.macos, JvmVersion.java8),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java10)),
```

Comment:
It's more of a question, but shouldn't we already run tests against 10 for the master branch too? Maybe we should define test coverage as follows:
- minimum and maximum Java version for `Master accept`
- all java versons for release accept.

/cc @oehme

Reply:
`Master Accept` and `Release Accept` or not branches, they are build stages. They are run both on `master` and `release`. `Release Accept` just means "everything looks good, we could release this if we wanted".

We could move Java 9 to `Release Accept` and instead do Java 10 in `Master accept`.

Reply:
I think we should both do Java 9/10 in `Master Accept`. Seems once a day is not enough to find out problems. Any objections?

Reply:
Once a day is enough for Java 9. We want to keep Master Accept fast, so please don't put too much in it. Java 10 should already cover most problems.